spyder/plugins/workingdirectory/container.py

```diff
@@ -78,7 +82,54 @@ def enterEvent(self, event):
         """Set current path as the tooltip of the widget on hover."""
         self.setToolTip(self.currentText())
 
+    def focusOutEvent(self, event):
+        """Handle focus out event restoring the last valid selected path."""
+        if self.add_current_text_if_valid():
+            self.selected()
+            self.hide_completer()
+        hide_status = getattr(self.lineEdit(), 'hide_status_icon', None)
+        if hide_status:
+            hide_status()
+        QComboBox.focusOutEvent(self, event)
```

Comment:
```suggestion
        super().focusOutEvent(self, event)
```

I think we should `super` here, right?